README.md

```diff
@@ -1,14 +1,67 @@
-# Compass
+![Compass logo](https://raw.githubusercontent.com/hyperoslo/Compass/update/readme/Images/logo_v1.png)
 
 [![CI Status](http://img.shields.io/travis/hyperoslo/Compass.svg?style=flat)](https://travis-ci.org/hyperoslo/Compass)
 [![Version](https://img.shields.io/cocoapods/v/Compass.svg?style=flat)](http://cocoadocs.org/docsets/Compass)
 [![License](https://img.shields.io/cocoapods/l/Compass.svg?style=flat)](http://cocoadocs.org/docsets/Compass)
 [![Platform](https://img.shields.io/cocoapods/p/Compass.svg?style=flat)](http://cocoadocs.org/docsets/Compass)
 
-## Usage
+Compass helps you setup a central navigation system for your application.
+This has many benefits, one of them being that controllers can now be
+decoupled, meaning that the list that presents the detail no longer knows 
```

Comment:
Maybe it's better to say that you could present any controller from any place in your app. The list and detail are just a specific example.